### PR TITLE
chore(deps): update to rc.5

### PR DESCRIPTION
**Pull request by Nightapes**
Update to rc.5 (https://github.com/mgechev/angular2-seed/issues/1207)

- add ngModules
- update tests
- fix e2e (downgraded to protractor 3.3.0, https://github.com/mgechev/angular2-seed/issues/1185)

### CODE REVIEW

**Comment by troyanskiy** [comment]
Fixes #1207

**Comment by ludohenin** [comment]
Awesome!

File: src/client/app/+about/about.module.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { NgModule }           from '@angular/core';
+import { CommonModule }       from '@angular/common';
+import{ AboutComponent }   from './about.component';
+
```
Comment (ludohenin) [inline src/client/app/+about/about.module.ts]: Can you fix the style here ?

File: src/client/app/+home/home.module.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { NgModule }           from '@angular/core';
+import { CommonModule }       from '@angular/common';
+import { SharedModule }        from '../shared/shared.module';
+import { HomeComponent }   from './home.component';
+import { NameListService }     from '../shared/name-list/index';
+
```
Comment (ludohenin) [inline src/client/app/+home/home.module.ts]: Same here ?

Comment (Nightapes) [inline src/client/app/+home/home.module.ts]: My formatter didn't worked, i fixed it.

**Comment by troyanskiy** [comment]
@mgechev Please merge it.

**Comment by mgechev** [comment]
Awesome job!

**Pull request merged**